pages/common/routers/form.js

```diff
@@ -1,15 +1,16 @@
 const { Router } = require('express');
 const bodyParser = require('body-parser');
-const { mapValues, size, some, get, isEqual, reduce, isUndefined, identity } = require('lodash');
+const { mapValues, size, some, get, isEqual, reduce, isUndefined, identity, pickBy } = require('lodash');
 const validator = require('../../../lib/validation');
 
 const defaultMiddleware = (req, res, next) => next();
 
-const hasChanged = (value, key, item) => {
-  if (Array.isArray(value)) {
-    return !isEqual([ ...value ].sort(), [ ...item[key] ].sort());
+const hasChanged = ({ accessor }, newValue, oldValue) => {
+  oldValue = get(oldValue, accessor, oldValue);
+  if (Array.isArray(newValue)) {
+    return !isEqual([ ...newValue ].sort(), [ ...oldValue ].sort());
```

Comment:
`.sort()` doesn't modify the array, so there's no particular reason for the `...`

Reply:
I just like the way they look

Reply:
wasn't part of this PR - but I'll update as I have the file open :)